### ISSUE TITLE
Ajax request are getting cancelled

### ISSUE THREAD

**Issue by Kaliver88** [nittro/nittro#28]
Hi,
I have list of an images as dynamics snippets. When I removed them very quickly I noticed a problem.
Every new ajax request cancels current request. Is there way to prevent it?

![canceled-requests](https://user-images.githubusercontent.com/29901194/37648470-5a3d2760-2c2f-11e8-9b52-85491691b7d4.png)

Thanks

**Comment by jahudka** [comment]
**TL;DR** - add `data-background="true"` to any links that you want to not be aborted when another link is clicked.

Hi, yeah, that's the default behaviour and it's actually by design - imagine that the user clicks three different menu items very quickly, so they all start loading almost simultaneously, but as each page may take a different amount of time to render on the server, the order the responses arrive at the client is undefined - how should the clientside code react to that?

So in trying to cover the 90% use-case scenario the Page service cancels any outstanding transaction as soon as a new one is initiated. And for the 10% there's the `background` option, either via `data-background="true"` or via the `context` parameter of `Page.open()`.
